index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Portofolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+             <ul>
+               <li><a class="myLogo" href="">Abdi Bekele</a> </li>
+               <li><a href="" class="toggler-icon"><img src="./assets/images/humberger.png" alt="humberger-icon"></a></li>
+
+             </ul>
+    
+
+
+
+        </nav>
+      
+    </header>
+       
+           
+       
+    
+    <section class= "headline">
+
+        <div class= "heyThere">
+           
+            <h1>Hey There, </h1>
+            <h1>I'm Abdi</h1></div>
+        
+        <div class="career">I am a Software Developer</div>
+        <div class="description"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></div>
+    <div class="links">
+        <ul>
+    <li><a href="#"><img src="./assets/images/github.svg"></a></li>
+  
+
+    <li><a href="#" class ="link"><img src="./assets/images/link.png" ></i></a></li>
+
+    <li><a href="#" class ="link"><img src="./assets/images/3rdLink.svg"></a></li>
+
+    <li><a href="#" class ="link"><img src="./assets/images/twitter4.svg"></a>></li>
```

Comment:
There is a > extra in this line of code and it is appearing in your social media links. Kindly remove it. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192077018-0a74746d-b192-48f4-b826-ca6004637613.png)